specification/quota/resource-manager/Microsoft.Quota/preview/2019-07-19-preview/quota.json

```diff
@@ -0,0 +1,609 @@
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "Deletes the QuotaRequest by name for the resources for the resource  provider at a specific location.",
+        "description": "For the specified location, deletes the quota request by request name for resources under the subscription. Delete will not cancel the Quota Request in progress. It will only delete the completed requests, whican be rejected or succeeded quota requests.",
+        "operationId": "QuotaRequests_Name_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Quota Request deleted."
+          },
+          "404": {
+            "description": "Error: Not found. Quota request for provided quotaRequest name not found"
+          },
+          "409": {
+            "description": "Error: Conflict. Quota request for provided quotaRequest is in processing state. Request cannot be deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Quota/providers/{providerId}/locations/{locationId}/quotarequests": {
+      "get": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "For the specified location and resource Provider, gets thecurrent quota requests under the subscription over the time\n period of one year ago from now to one year back.",
+        "description": "For the specified location and Resource provider gets the current quota requests under the subscription over the time period of one year ago from now to one year back. oData filter can be used to select quota requests.",
+        "operationId": "QuotaRequest_status",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Sending Quota request status and details",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestDetailsByNameList"
```

Comment:
`QuotaRequestDetailsList`, unless there are really different lists based on how they were retrieved.

Reply:
QuotaRequestDetailsByName - Include the QuotaRequest Name also.
QuotaRequestDetail is response for - Get /subscriptions/{subscriptionId}/providers/Microsoft.Quota/providers/{providerId}/locations/{locationId}/quotaRequests/{requestName}
The request name is already specified in the request, so it's not included in the response.
Also changed QuotaRequestDetails to QuotaRequestDetail .
Removed QuotaRequestDetails

Reply:
Please review the changes and comments. I have addressed the questions and updted the API.

Reply:
Removed the error word from the response.